src/token/Tranche.sol

```diff
@@ -32,8 +33,13 @@ contract TrancheToken is ERC20 {
 
     mapping(address => bool) public trustedForwarders;
 
+    /// @dev Look up vault by the asset (part of ERC7575)
+    mapping(address asset => address) public vault;
```

Comment:
I'm not a huge fan of referring to liquidity pools by another name here (although I understand its a requirement for the ERC). Should we consider renaming liquidity pools to vaults?